server/api/contact.post.ts

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
I assume we will wait for this to merge?

Reply:
yes, I wasn't sure how you manage contact form

Reply:
so I think you can delete this file because I sent a test message on your netlify deploy and it works 🤷‍♀️ 

Must be some netlify magic

<img width="418" alt="image" src="https://github.com/devedmonton/DES-Website/assets/17459171/9c9d64db-a4a1-4fb8-bc88-26ebfe918f0c">

Reply:
delete this file